cmd/config/certs.go

```diff
@@ -82,22 +82,21 @@ func GetRootCAs(certsCAsDir string) (*x509.CertPool, error) {
 
 	fis, err := ioutil.ReadDir(certsCAsDir)
 	if err != nil {
-		if os.IsNotExist(err) {
-			err = nil // Return success if CA's directory is missing.
+		if os.IsNotExist(err) || os.IsPermission(err) {
+			// Return success if CA's directory is missing or permission denied.
+			err = nil
 		}
 		return rootCAs, err
 	}
 
 	// Load all custom CA files.
 	for _, fi := range fis {
-		// Only load regular files as public cert.
-		if fi.Mode().IsRegular() {
-			caCert, err := ioutil.ReadFile(filepath.Join(certsCAsDir, fi.Name()))
-			if err != nil {
-				return rootCAs, err
-			}
-			rootCAs.AppendCertsFromPEM(caCert)
+		caCert, err := ioutil.ReadFile(path.Join(certsCAsDir, fi.Name()))
```

Comment:
Hm, on windows that will break since the `path` package uses `/` and windows separates directors with `\`.
We should `filepath` not `path` here.

Reply:
It won't @aead Go automatically handles it, also Win32 APIs handle both styles.